components/header/header.module.scss

```diff
@@ -0,0 +1,112 @@
+@import '../../styles/variables';
+@import '../../styles/mixins';
+
+.container {
+  background: $color-dark;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  z-index: 5;
+  margin-bottom: 100px;
+}
+
+.header {
+  background: $color-dark;
+  display: flex;
+  height: $header-height;
+  justify-content: space-between;
+  margin: auto;
+  max-width: $max-content;
+  position: sticky;
+  text-align: center;
+
+  @include breakpoint(small) {
+    padding: 0.5rem;
+  }
+}
+
+.logo {
+  display: flex;
+
+  @include breakpoint(medium-only) {
+    margin-left:3.5rem;
```

Comment:
Try and have a space after the `:` here. This is something that should be picked up automatically by prettier when you save the file but also when things are committed or pushed. Maybe that isn't working as it should. I'll investigate.

Reply:
this is because i tried to resolve conflicts on github. i am fixing it now